### ISSUE TITLE
PHP Fatal error when running against my project

### ISSUE THREAD

**Issue by thekabal** [slevomat/coding-standard#116]
Installed using:

            "slevomat/coding-standard": "^2.0",
            "consistence/coding-standard": "^0.13",

and php composer.phar update / php composer.phar install.

Then edited vendor/bin/phpcs.xml to add:
```
    <rule ref="vendor/consistence/coding-standard/Consistence/ruleset.xml" />
    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/ruleset.xml" />
```

Not sure I understand the nature of the error, but its entirely possible that I installed wrong, am running it incorrectly, or that my project (with some code from php-4 days) is triggering an unusual corner case.

```
$ php vendor/bin/phpcs --standard=vendor/bin/phpcs.xml . --ignore=templates,vendor
PHP Fatal error:  Uncaught SlevomatCodingStandard\Sniffs\Namespaces\NoKeywordsException: Sniff SlevomatCodingStandard\Sniffs\Namespaces\FullyQualifiedClassNameAfterKeywordSniff requires an array of keywords set in property keywordsToCheck in /var/www/html/tki/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameAfterKeywordSniff.php:42
Stack trace:
#0 /var/www/html/tki/vendor/squizlabs/php_codesniffer/CodeSniffer.php(1480): SlevomatCodingStandard\Sniffs\Namespaces\FullyQualifiedClassNameAfterKeywordSniff->register()
#1 /var/www/html/tki/vendor/squizlabs/php_codesniffer/CodeSniffer.php(595): PHP_CodeSniffer->populateTokenListeners()
#2 /var/www/html/tki/vendor/squizlabs/php_codesniffer/CodeSniffer/CLI.php(956): PHP_CodeSniffer->initStandard(Array, Array, Array)
#3 /var/www/html/tki/vendor/squizlabs/php_codesniffer/CodeSniffer/CLI.php(113): PHP_CodeSniffer_CLI->process()
#4 /var/www/html/tki/vendor/squizlabs/php_codesniffer/scripts/phpcs(25): PHP_CodeSniffer_CLI->runphpcs()
#5 {main}
   in /var/www/html/tki/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameAfterKeywordSniff.php on line 42
```

**Comment by ondrejmirtes** [comment]
You need to configure the sniff as said in the error message:

```
Sniff SlevomatCodingStandard\Sniffs\Namespaces\FullyQualifiedClassNameAfterKeywordSniff requires an array of keywords set in property keywordsToCheck
```

**Comment by ondrejmirtes** [comment]
You can either exclude the sniff (if you don't want fully qualified names after any keyword) or configure its property.

**Comment by thekabal** [comment]
How do I exclude the hint OR configure the keywords?

**Comment by ondrejmirtes** [comment]
https://github.com/slevomat/coding-standard#using-individual-sniffs

And configuring this exact sniff is described in the README: https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesfullyqualifiedclassnameafterkeyword

**Comment by thekabal** [comment]
That worked! Many thanks!!
